src/index.ts

```diff
@@ -0,0 +1,19 @@
+import IDecoratedNumber from "./decorators/decorated-number";
+import decorators from "./decorators/index";
+import detectCountry from "./detect-country";
+
+export default {
+  decorate: (phoneNumber: string): IDecoratedNumber => {

```

Comment:
It's assumed that this is an unformatted number? For instance `61023333444` (from the tests). Would we want to do any basic whitespace stripping or removing a trailing `+`?

Reply:
Yeah, we will strip out any non-numeric characters before we call decorate. I'm saving that for a future PR though.

There's a lot of logic around the way people specify international dialing, as well as local leading digits and what not.